ui/pages/snap-account-redirect/snap-account-redirect.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import SnapAccountRedirect from './snap-account-redirect';
+
+export default {
+  title: 'Components/UI/SnapAccountRedirect',
+} as Meta<typeof SnapAccountRedirect>;
+
+export const DefaultStory = () => (
+  <SnapAccountRedirect
+    snapName="Snap Simple Keyring"
+    isBlockedUrl={false}
+    url="https://metamask.github.io/snap-simple-keyring/0.2.4/"
+    message="Redirecting to Snap Simple Keyring"
+  />
+);
+
+DefaultStory.storyName = 'Default';
```

Comment:
You can make the story more interactive by moving the data to args and allowing the controls to be used

```suggestion
  argTypes: {
    snapName: {
      control: {
        type: 'text',
      },
    },
    isBlockedUrl: {
      control: {
        type: 'boolean',
      },
    },
    url: {
      control: {
        type: 'text',
      },
    },
    message: {
      control: {
        type: 'text',
      },
    },
  },
  args: {
    snapName: 'Snap Simple Keyring',
    isBlockedUrl: false,
    url: 'https://metamask.github.io/snap-simple-keyring/0.2.4/',
    message: 'Redirecting to Snap Simple Keyring',
  },
} as Meta<typeof SnapAccountRedirect>;

export const DefaultStory = (args) => <SnapAccountRedirect {...args} />;

DefaultStory.storyName = 'Default';

```

Reply:
Changed on 523c7575f9119e48914b5059d4e2b53f6c5b2a3f